tests/components/renault/test_binary_sensor.py

```diff
@@ -0,0 +1,195 @@
+"""Tests for Renault binary sensors."""
+from unittest.mock import patch
+
+import pytest
+from renault_api.kamereon import exceptions
+
+from homeassistant.components.binary_sensor import DOMAIN as BINARY_SENSOR_DOMAIN
+from homeassistant.const import (
+    ATTR_IDENTIFIERS,
+    ATTR_MANUFACTURER,
+    ATTR_MODEL,
+    ATTR_NAME,
+    ATTR_SW_VERSION,
+    STATE_OFF,
+    STATE_UNAVAILABLE,
+)
+from homeassistant.setup import async_setup_component
+
+from . import (
+    setup_renault_integration_vehicle,
+    setup_renault_integration_vehicle_with_no_data,
+    setup_renault_integration_vehicle_with_side_effect,
+)
+from .const import MOCK_VEHICLES
+
+from tests.common import mock_device_registry, mock_registry
+
+
+@pytest.mark.parametrize("vehicle_type", MOCK_VEHICLES.keys())
+async def test_binary_sensors(hass, vehicle_type):
+    """Test for Renault binary sensors."""
+    await async_setup_component(hass, "persistent_notification", {})
+    entity_registry = mock_registry(hass)
+    device_registry = mock_device_registry(hass)
+
+    with patch("homeassistant.components.renault.PLATFORMS", [BINARY_SENSOR_DOMAIN]):
+        await setup_renault_integration_vehicle(hass, vehicle_type)
+        await hass.async_block_till_done()
+
+    mock_vehicle = MOCK_VEHICLES[vehicle_type]
+    expected_entities = mock_vehicle[BINARY_SENSOR_DOMAIN]
+    if len(expected_entities) == 0:
```

Comment:
```suggestion
    if vehicle_type == "captur_fuel":
```
This way it's not dynamic, and will cause issues when/if the fixture changes